src/render/shaders/tilemap.wgsl

```diff
@@ -34,8 +34,9 @@ var<uniform> tilemap_data: TilemapData;
 #include 0
 
 struct VertexOutput {
-    @location(0) uv: vec3<f32>,
+    @location(0) uv: vec2<f32>,
     @location(1) color: vec4<f32>,
+    @location(2) @interpolate(flat) tile_id: f32,
```

Comment:
Optionally we could just use a u32 here. 🤷

Reply:
yeah why not use integers? that seems like the best way to ensure everything is correct